.github/workflows/merge-strategy.yml

```diff
@@ -0,0 +1,17 @@
+name: Merge Strategy
```

Comment:
this stuff should be in a separate PR ideally

Reply:
forgot to rebase off main, didn't mean to include this